mindsdb/integrations/handlers/hubspot_handler/hubspot_handler.py

```diff
@@ -49,48 +50,501 @@ def __init__(self, name: str, **kwargs):
+
+        try:
+            ast = parse_sql(query)
+            return self.query(ast)
+        except Exception as e:
+            logger.error(f"Failed to execute native query: {str(e)}")
+            return Response(RESPONSE_TYPE.ERROR, error_message=f"Query execution failed: {str(e)}")
+
+    def get_tables(self) -> Response:
+        """Return list of tables available in the HubSpot integration.
+
+        Returns:
+            Response: A response containing table metadata including table names, types,
+                     estimated row counts, and descriptions.
+        """
+        try:
+            self.connect()
+
+            # Get basic table information
+            tables_data = []
+
+            for table_name in ["companies", "contacts", "deals"]:
+                try:
+                    table_info = {
+                        "TABLE_NAME": table_name,
+                        "TABLE_TYPE": "BASE TABLE",
+                        "TABLE_SCHEMA": "hubspot",
+                        "TABLE_DESCRIPTION": self._get_table_description(table_name),
+                        "ROW_COUNT": self._estimate_table_rows(table_name),
+                    }
+                    tables_data.append(table_info)
+
+                except Exception as e:
+                    logger.warning(f"Could not get metadata for table {table_name}: {str(e)}")
+
+                    tables_data.append(
+                        {
+                            "TABLE_NAME": table_name,
+                            "TABLE_TYPE": "BASE TABLE",
+                            "TABLE_SCHEMA": "hubspot",
+                            "TABLE_DESCRIPTION": self._get_table_description(table_name),
+                            "ROW_COUNT": None,
+                        }
+                    )
+
+            df = pd.DataFrame(tables_data)
+            logger.info(f"Retrieved metadata for {len(tables_data)} tables")
+            return Response(RESPONSE_TYPE.TABLE, data_frame=df)
+
+        except Exception as e:
+            logger.error(f"Failed to get tables: {str(e)}")
+            return Response(RESPONSE_TYPE.ERROR, error_message=f"Failed to retrieve table list: {str(e)}")
+
+    def get_columns(self, table_name: str) -> Response:
+        """Return column information for a specific table.
+
+        Args:
+            table_name (str): Name of the table to get column information for
+
+        Returns:
+            Response: A response containing column metadata including names, types,
+                     descriptions, and statistics.
+        """
+        if table_name not in ["companies", "contacts", "deals"]:
+            return Response(
+                RESPONSE_TYPE.ERROR,
+                error_message=f"Table '{table_name}' not found. Available tables: companies, contacts, deals",
+            )
+
+        try:
+            self.connect()
+
+            columns_data = self._get_columns_with_statistics(table_name)
```

Comment:
We show this columns in table `information_schema.columns`. Columns list should be the same as in the `information_schame.columns` table, statistic there is not necessary. Check [here](https://github.com/mindsdb/mindsdb/blob/main/mindsdb/integrations/libs/response.py#L74) and [here ](https://github.com/mindsdb/mindsdb/blob/main/mindsdb/integrations/handlers/postgres_handler/postgres_handler.py#L460)for reference